libexec/python/docker/api.py

```diff
@@ -26,7 +26,7 @@
 import os
 sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
                                 os.path.pardir)))  # noqa
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) # noqa
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))  # noqa
```

Comment:
*and then tiny robot cheers erupted from the peanut gallery, and the pep8 space was fixed, and all was well!*